src/RetargetingHelper/src/Helper.cpp

```diff
@@ -144,21 +360,104 @@ const iDynTree::Transform& RetargetingClient::rightHandTransform() const
     return m_rightHandTransform;
 }
 
+const iDynTree::VectorDynSize& RetargetingClient::jointValues() const
+{
+    return m_jointRetargetingValue;
```

Comment:
How does the consumer class know if this output is meaningful or not? If I understood correctly, the hand retargeting and the joint retargeting cannot be enabled at the same time. Hence, how can you know if the value contained in this vector is garbage or not?

Reply:
we may return an `std::pair<bool, const iDynTree::VectorDynSize&>`. I don't really like it, but it could be a way to solve the problem.

Reply:
I agree, maybe adding a method like ``isJointRetargetingEnabled`` or whatever name you like? Do you think it may be an option?

Reply:
I don't remember if something changed here :thinking:

Reply:
Even if the joint retargeting is not enable the data are meaningful because `m_jointRetargetingValue` is reset using the initial position of the robot joints.

Reply:
Ok great!.